README.md

```diff
@@ -615,8 +615,12 @@ may also be specified using the options dict parameter.
 
 Possible `options`:
 
-* data_format - must be a string, choices: "dataframe", "records", "json"
+* data_format - must be a string, choices: "dataframe", "records", "json", "data_stream"
```

Comment:
I'm not sure `data_stream` is clear, `payload` or something like that?

Reply:
Switching to flattened_dataframe because that is what it is